catalyst_voices/packages/catalyst_voices_services/lib/src/registration/registration_service.dart

```diff
@@ -1,9 +1,46 @@
+import 'dart:math';
+
 import 'package:catalyst_cardano/catalyst_cardano.dart';
 import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
 import 'package:catalyst_voices_models/catalyst_voices_models.dart';
 import 'package:catalyst_voices_repositories/catalyst_voices_repositories.dart';
 import 'package:catalyst_voices_services/src/registration/registration_transaction_builder.dart';
 
+// TODO(damian-molinski): remove once recover account is implemented
+final _testNetAddress = ShelleyAddress(
+  const [
```

Comment:
the byte list representation is extremely uncommon because it's unreadable, most common is bech32 encoded, like this:

```
ShelleyAddress.fromBech32(
    'addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw',
  )
```

Reply:
tied it but then CI was sad about typos and i didn't want to put entire string into dictionary

Reply:
I often do:

```
              /* cSpell:disable */
              walletAddress: ShelleyAddress.fromBech32(
                'addr_test1vzpwq95z3xyum8vqndgdd'
                '9mdnmafh3djcxnc6jemlgdmswcve6tkw',
              ),
              /* cSpell:enable */
```

Reply:
Maybe this would also work:

```
// cSpell:disable
something that breaks cspell
// cSpell:enable
```

Reply:
fixed